build/ScaffoldCodeFix.fs

```diff
@@ -98,9 +63,29 @@ let fix
       let! (parseAndCheckResults:ParseAndCheckResults, line:string, sourceText:IFSACSourceText) =
         getParseResultsForFile fileName fcsPos
 
-      // As an example, we want to check whether the users cursor is inside a function definition name.
-      // We will traverse the syntax tree to verify this is the case.
-      match visitSyntaxTree fcsPos parseAndCheckResults.GetParseResults.ParseTree with
+      // The syntax tree can be an intimidating set of types to work with.
+      // It is a tree structure but it consists out of many different types.
+      // See https://fsharp.github.io/fsharp-compiler-docs/reference/fsharp-compiler-syntax.html
+      // It can be useful to inspect a syntax tree via a code sample using https://fsprojects.github.io/fantomas-tools/#/ast
+      // For example `let a b c = ()` in
+      // https://fsprojects.github.io/fantomas-tools/#/ast?data=N4KABGBEAmCmBmBLAdrAzpAXFSAacUiaAYmolmPAIYA2as%%2BEkAxgPZwWQ2wAuYVYAEZhmYALxgAFAEo8BSLAAeAByrJoFHgCcArrBABfIA
+      // Let's say we want to find the (FCS) range for identifier `a` if the user's cursor is inside the function name.
+      // We will query the syntax tree to verify this is the case.
+      let maybeFunctionNameRange =
+        (fcsPos, parseAndCheckResults.GetParseResults.ParseTree)
+        ||> ParsedInput.tryPick (fun path node ->
```

Comment:
When you scaffold a new code fix, the `path` parameter here will be unused, which will result in FS1182 "The value 'path' is unused."

Is it OK to leave it to the implementer to decide whether they need to use `path` in their logic or to ignore it?

Or should we have `_path` in the example so that the warning (which is treated as an error here) isn't triggered?

Reply:
`_path` is fine by me

Reply:
82eba4f